doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -167,133 +165,74 @@ Enable partition mode in ACRN hypervisor
 #. Configure the PCI device info for each VM
 
    PCI devices that are available to the privileged VMs
-   are hardcoded in the source file ``hypervisor/partition/apl-mrb/vm_description.c``.
-   You need to review and modify the ``vpci_vdev_array1`` and ``vpci_vdev_array2``
-   structures in the source code to match the PCI BAR addresses of the SATA
+   are hardcoded in the source file ``hypervisor/arch/x86/configs/apl-mrb/pt_dev.c``.
+   You need to review and modify the ``vm0_pci_ptdevs`` and ``vm1_pci_ptdevs``
+   structures in the source code to match the PCI BDF addresses of the SATA
    controller and the USB controller noted in step 1:
 
    .. code-block:: none
-     :emphasize-lines: 3,39,44,49,56,72
-     :caption: hypervisor/partition/vm_description.c
+     :emphasize-lines: 5,9,17,21,25
```

Comment:
I can see why it makes sense to highlight lines 9 and 21 (where the BDF collected above are used) but what is the meaning of the other lines highlighted?

I guess, looking at the current version of that doc, that the first BDF (`0:0.0`) is for the host bridge. Can we add this in the code like it used to be)? It would make things clearer.

I'm also assuming that we can use any BDF for the `vbdf.bits` as long as it hasn't been used yet. If true, some wording on this would be useful.